libraries/AP_GPS/AP_GPS_UBLOX.h

```diff
@@ -101,6 +103,8 @@ class AP_GPS_UBLOX : public AP_GPS_Backend
 
     static bool _detect(struct UBLOX_detect_state &state, uint8_t data);
 
+    bool supports_mavlink_gps_rtk_message() { return true; }
```

Comment:
This should be made conditional on actually having received some none 0 information into the state. It's otherwise quite a waste of MAVLink bandwidth for most users that unless you are receiving the packet from the GPS (and potentially only if receiving it + actually in a RTK fix state) should it be sent.